src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,121 @@
 import React from 'react';
 import './TodoList.scss';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
 
-export const TodoList = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
-          >
-            User&nbsp;#2
-          </button>
-        </li>
-      </ul>
-    </div>
-  </div>
-);
+export class TodoList extends React.Component {
+  state = {
+    query: '',
+    filteredBy: '',
+  }
+
+  handleChange = (event) => {
+    const { name, value } = event.target;
+
+    this.setState({
+      [name]: value,
+    });
+  }
+
+  render() {
+    const { todos, selectedUser, todoCheck } = this.props;
+    const input = this.state.query.toLocaleLowerCase();
+
+    const preparedTodos = todos.filter(todo => todo.title.toLocaleLowerCase()
+      .includes(input))
+      .filter((todo) => {
+        if (this.state.filteredBy === 'true') {
+          return todo.completed === true;
+        }
+
+        if (this.state.filteredBy === 'false') {
+          return todo.completed === false;
+        }
+
+        return todo;
+      });
```

Comment:
switch case

Reply:
also let's move this to the separate function filterByStatus and use it like this
`.this.filter(this.filterByStatus)`